setup.py

```diff
@@ -33,6 +33,7 @@
       author_email='peter@svinota.eu',
       url='https://github.com/svinota/pyroute2',
       license='dual license GPLv2+ and Apache v2',
+      install_reqiures=['psutil~=5.0'],
```

Comment:
```suggestion
      install_requires=['psutil~=5.0'],
```

Reply:
there could be a breaking change on the 6.0 release, we would want an upper bound